storage/mysql/68-mysql.js

```diff
@@ -122,7 +122,24 @@ module.exports = function(RED) {
                 if (node.mydbConfig.connected) {
                     if (typeof msg.topic === 'string') {
                         //console.log("query:",msg.topic);
-                        var bind = Array.isArray(msg.payload) ? msg.payload : [];
+                        var bind = [];
+                        if(Array.isArray(msg.payload)){bind = msg.payload}
```

Comment:
we prefer spaces `if (`  and `) { ` for readability

Reply:
I've changed